tests/integration/targets/vmware_host_iscsi/tasks/reset_iscsi_config_tasks.yml

```diff
@@ -0,0 +1,41 @@
+# Test code for the vmware_host_logbundle module.
+# Copyright: (c) 2020, sky-joker <sky.jokerxx@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- name: reset iSCSI config tasks
+  block:
+    - vmware_host_iscsi:
+        hostname: "{{ vcenter_hostname }}"
+        username: "{{ vcenter_username }}"
+        password: "{{ vcenter_password }}"
+        validate_certs: no
+        esxi_hostname: "{{ esxi1 }}"
+        iscsi_config:
+          alias: ''
+          vmhba_name: "{{ vmhba_name }}"
+        state: present
+
+    - vmware_host_iscsi:
+        hostname: "{{ vcenter_hostname }}"
+        username: "{{ vcenter_username }}"
+        password: "{{ vcenter_password }}"
+        validate_certs: no
+        esxi_hostname: "{{ esxi1 }}"
+        iscsi_config:
+          port_bind:
+            - vmk0
+          vmhba_name: "{{ vmhba_name }}"
+          send_target:
+            address: 100.64.0.1
```

Comment:
I'm a but confused because you call the file `reset_iscsi_config_tasks.yml `, but you actually set some new configuration.

Reply:
Tasks performed by `reset_iscsi_config_tasks` have been moved to`connect_to_vcenter_tasks.yml` and `connect_to_esxi_tasks.yml`.
Because of that, I have removed reset_iscsi_config_tasksl.yml file.